### ISSUE TITLE
version 3.1.25 causes timeout

### ISSUE THREAD

**Issue by xavierpriour** [smarty-php/smarty#56]
Hi,
My code works in 3.1.24, but gets the following error in 3.1.25:
```
[Wed Jun 17 10:45:31 2015] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in
 /Users/Xavier/xProjects/time-track/.dist/local/20150617T084453Z/vendor/
smarty/smarty/libs/sysplugins/smarty_internal_resource_file.php on line 131
[Wed Jun 17 10:45:31 2015] 127.0.0.1:58693 [500]: /php/login.php - Maximum execution time of 30 seconds exceeded in
/Users/Xavier/xProjects/time-track/.dist/local/20150617T084453Z/vendor/
smarty/smarty/libs/sysplugins/smarty_internal_resource_file.php on line 131
```

My code is:
- in `initialize.php`
```php
$smarty = new Smarty();
$smarty->use_sub_dirs = false;
$smarty->setTemplateDir("$lib/smarty/templates");
$smarty->setCompileDir("$lib/smarty/templates_c");
$smarty->addPluginsDir("$lib/smarty/plugins");
```
- in `login.php`, I require_once `initialize.php` then
```php
// lots of $smarty->assign
$smarty->display('login.tpl');
```

I have replaced `login.tpl` with a very basic html text (no php, no tags), and I still get the timeout.
Any idea what could be the issue?

Also, the smarty.net site seems down

**Comment by uwetews** [comment]
How does $lib look like?  Does it contain "//." or "\\."?

**Comment by xavierpriour** [comment]
definition is `$lib = realpath(dirname(__FILE__) . '/..');`

But `var_dump($lib)` prints out `"/home/xavier/xProjects/time-track/.dist/local/20150617T210630Z/lib"` 

The calling page `login.php` is located in `"/home/xavier/xProjects/time-track/.dist/local/20150617T210630Z/www/php"`

**Comment by uwetews** [comment]
The fix is now in release 3.1.27
